example.html

```diff
@@ -6,7 +6,7 @@
     <title>Pure CSS gauge v0.2</title>
     <meta name="description" content="Pure CSS gauge">
     <meta name="author" content="Otto Salminen">
-    <link rel="stylesheet" href="gauge.min.css?v=0.2">
```

Comment:
Would use the minified version on the default example

Reply:
Minified version updated and reverted to it here :)